core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/WorkspaceManagerResourceMonitorRecordDao.scala

```diff
@@ -0,0 +1,32 @@
+package org.broadinstitute.dsde.rawls.dataaccess
+
+import org.broadinstitute.dsde.rawls.dataaccess.slick.WorkspaceManagerResourceMonitorRecord
+
+import java.sql.Timestamp
+import java.time.Instant
+import java.util.UUID
+import scala.concurrent.Future
+
+class WorkspaceManagerResourceMonitorRecordDao(val dataSource: SlickDataSource) {
+
+  def create(jobRecordId: UUID, billingProjectName: String): Future[Unit] = create(
+    WorkspaceManagerResourceMonitorRecord(
+      jobRecordId,
+      WorkspaceManagerResourceMonitorRecord.JobType.AzureLandingZoneResult,
```

Comment:
We don't want to hardcode LandingZone result here.

Reply:
I think I just moved that from the datasource component file without changing it (time to change it)